src/main/java/com/synopsys/integration/alert/channel/jira/actions/JiraDistributionTestAction.java

```diff
@@ -83,7 +84,7 @@ public String testConfig(String jobId, String destination, FieldAccessor fieldAc
             toStatus = "Resolve";
             logger.debug("Sending additional DELETE test message...");
             final DistributionEvent reResolveIssueEvent = createChannelTestEvent(jobId, fieldAccessor, ItemOperation.DELETE, messageId);
-            String reResolveResult = getDistributionChannel().sendMessage(reResolveIssueEvent);
+            MessageResult reResolveResult = getDistributionChannel().sendMessage(reResolveIssueEvent);
```

Comment:
Why do we only care about the initial and the last results?

Reply:
The initial results let us know that the issue was created. The last results let us know that we made it all the way to the end without erroring out.